.github/workflows/tm.integration.tests.yml

```diff
@@ -52,3 +52,10 @@ jobs:
         uses: ./.github/actions/tm-integration-tests
         env:
           GOPATH: /github/workspace
+      - name: go-version
+        run: echo "::set-output name=value::$(cat GO_VERSION)"
+      - name: Install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ steps.go-version.outputs.value }}
```

Comment:
Setting up Go needs to come before the *Run integration tests* step.

Reply:
Moved to run before integration test.